src/Microsoft.SqlTools.ServiceLayer/sr.resx

```diff
@@ -13,16 +13,16 @@
     Example:
     
     ... ado.net/XML headers & schema ...
-    <resheader name="resmimetype">text/microsoft-resx</resheader>
-    <resheader name="version">2.0</resheader>
-    <resheader name="reader">System.Resources.ResXResourceReader, System.Windows.Forms, ...</resheader>
-    <resheader name="writer">System.Resources.ResXResourceWriter, System.Windows.Forms, ...</resheader>
-    <data name="Name1"><value>this is my long string</value><comment>this is a comment</comment></data>
-    <data name="Color1" type="System.Drawing.Color, System.Drawing">Blue</data>
-    <data name="Bitmap1" mimetype="application/x-microsoft.net.object.binary.base64">
+    <resheader name="resmimetype=">text/microsoft-resx</resheader>
```

Comment:
I'm kind of confused as to how these '=' characters got added here, was this changed by hand or is there a formatting issue with the sr.strings file?